Automattic/jetpack
# Make Backup plugin more clear about realtime backups.

This PR updates the the text in the Backup standalone dashboard, making it more clear that backups are triggered in real-time as the site is edited or changed.
Also added an additional link pointing to the site's backup & restore points in the wordpress.com Activity Log.

See detailed changes in this i2 design P2: p1HpG7-gkG-p2#visuals

**Additional note:** Page-view and link tracking (Tracks) will be implemented in a following PR. ( #24998 ) 

### Changes proposed in this Pull Request:
**BEFORE:**

![Screenshot on 2022-07-05 at 09-11-13](https://user-images.githubusercontent.com/11078128/177335713-5cf59c81-fb73-4aaf-8e75-bf33e4e69a85.png)

**AFTER:**

**_Desktop:_**
![Markup 2022-07-06 at 17 03 16](https://user-images.githubusercontent.com/11078128/177642803-75d9c982-d445-4168-8d23-9533d544a8a8.png)

.
**_Mobile_**

<img src="https://user-images.githubusercontent.com/11078128/177648832-c567e9f0-0ce1-44a4-87ec-de30a920e188.png" alt="backup-screenshot-mobile" width="450">
----------------------------------------------------------------

### Other information:


### Jetpack product discussion

Project Thread: p1HpG7-gb4-p2

All changes are further described in this i2 design P2: p1HpG7-gkG-p2#visuals

Related to: https://github.com/Automattic/jetpack/pull/24919

### Does this pull request change what data or activity we track or use?


No

### Testing instructions:

- Spin up a JN site with Jetpack Beta, using Backup branch `update/backup-standalone-update-copy`.  
- Connect Backup and purchase Backup plan (with credits).
- Go to `https://cloud.jetpack.com/settings/:site?action=edit` (replace `:site` with your JN site slug) and enter your **JN site server credentials**.
- Navigate to the Jetpack --> Backup page (`/wp-admin/admin.php?page=jetpack-backup`).
- Verify the Backup dashboard page copy, text, and links looks like the design shown in p1HpG7-gkG-p2#visuals (there is also the Figma link in that post).
- Verify the page and copy looks all good in mobile viewport sizes.
- Check that all the links are working and they are pointing to the correct destinations.

**Additional note:** Page-view and link tracking (Tracks) will be implemented in a following PR.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202584524353834
